### PR TITLE
M16735 - Replying to an older bot post that has no post content and no attachment pretext

### PR DESCRIPTION
**Pull request by lawikip**
#### Summary
UI Automation: Write an automated test using Cypress for "Replying to an older bot post that has no post content and no attachment pretext"

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-16735

#### Screenshots
image

### CODE REVIEW

**Comment by lawikip** [comment]
@josephbaylon: - ready fo review

File: e2e/cypress/integration/channel/message_reply_no_pretext_spec.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users.json';
+
+const sysadmin = users.sysadmin;
+
+describe('Message Reply to bot message', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel/message_reply_no_pretext_spec.js]:
```suggestion
describe('Messaging', () => {
```

File: e2e/cypress/integration/channel/message_reply_no_pretext_spec.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    before(() => {
+        // # Set ServiceSettings to expected values
+        const newSettings = {
+            ServiceSettings: {
+                EnableBotAccountCreation: true,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Login and go to /
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel/message_reply_no_pretext_spec.js]:
```suggestion
        // # Login as sysadmin
```

File: e2e/cypress/integration/channel/message_reply_no_pretext_spec.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        cy.apiLogin('sysadmin');
+
+        // # Create and visit new channel
+        cy.createAndVisitNewChannel().then((channel) => {
+            newChannel = channel;
+        });
+    });
+
+    it('MM-16735 Reply to an older bot post that has no content and no attachment pretext', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel/message_reply_no_pretext_spec.js]:
```suggestion
    it('M16735 - Replying to an older bot post that has no post content and no attachment pretext', () => {
```

Comment (saturninoabril) [reply e2e/cypress/integration/channel/message_reply_no_pretext_spec.js]: For the [other test](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/channel/message_reply_pretext_spec.js#L35), could please change test description to "M16734 - Replying to an older bot post that has some attachment pretext before the message attachment"?

**Comment by saturninoabril** [comment]
nit: not really an issue but I can see that this PR includes unnecessary huge number of commits.  It's because you're working on master and other commits are from your previous submission (which was also worked on in master). In the future, it would be nice if you have it on a separate branch.

**Pull request closed without merging**